src/UserForm.js

```diff
@@ -246,11 +253,36 @@ class UserForm extends React.Component {
   }
 
   executeSave() {
-    const { handleSubmit, onSubmit } = this.props;
+    const {
+      handleSubmit,
+      onSubmit,
+    } = this.props;
+
     const submitter = handleSubmit(onSubmit);
+
     submitter();
   }
 
+  getActionMenu = ({ onToggle }) => {
+    const { onCancel } = this.props;
+    const handleClick = () => {
```

Comment:
Isn't it better to move this function to a separate method?

Reply:
anyway it's impossible to avoid creating of a new function

Reply:
Ok, I see